src/EditorFeatures/CSharp/SplitStringLiteral/SplitStringLiteralCommandHandler.cs

```diff
@@ -99,7 +99,8 @@ public bool ExecuteCommandWorker(ReturnKeyCommandArgs args)
             }
 
             var parsedDocument = ParsedDocument.CreateSynchronously(document, CancellationToken.None);
-            var options = subjectBuffer.GetIndentationOptions(_editorOptionsService, parsedDocument.LanguageServices, explicitFormat: false);
+            var formattingOptions = subjectBuffer.GetLineFormattingOptions(_editorOptionsService, explicitFormat: false);
```

Comment:
>formattingOptions [](http://example.com/codeflow?start=16&length=17)

The formatting options are already included in IndentationOptions

Reply:
it is not afaict :)

Reply:
`IndentationOptions.FormattingOptions.LineFormatting`?

Reply:
d'oh.  i'm dumb.  fixing.